BuyPolish/PolaUITests/Tests/ScanBarcodeWithResultPageUITests.swift

```diff
@@ -56,4 +56,20 @@ final class ScanBarcodeWithResultPageUITests: PolaUITestCase {
 
         XCTAssert(result, "Application rating alert did not appear")
     }
+
+    func testLongName() {
```

Comment:
Ten test powinien być w klasie `CompanyPageUITests`. Nagrany snapshot możesz przenieść między katalogami to nie będzie potrzeby nagrywać na nowo.